3rdtools.sh

```diff
@@ -68,7 +68,7 @@ function install {
   
   pushd glog-0.4.0
   clean_exec ./autogen.sh
-  clean_exec CXXFLAGS='-fPIC' CFLAGS='-fPIC' ./configure --enable-shared=yes --enable-static=yes --prefix=${rootdir}/3rd/glog-0.4.0
+  clean_exec CXXFLAGS='-fPIC' CFLAGS='-fPIC' ./configure --enable-shared=yes --enable-static=yes --prefix=${rootdir}/3rd/glog-0.4.0 --with-gflags=${rootdir}/3rd/gflags
```

Comment:
With this optoin, build all example will got below error.

```
BAZEL_LINKOPTS=-static-libstdc++ CC=/opt/mpich-3.2.1/bin/mpicxx bazel build --verbose_failures example/...
```

ERROR: /home/ced/opensource/plato/plato/util/BUILD:455:1: C++ compilation of rule '//plato/util:thread_local_object' failed (Exit 1) mpicxx failed: error executing command 
  (cd /home/ced/.cache/bazel/_bazel_ced/d3af6ab933838aa4af081ccbbafd9c2e/sandbox/processwrapper-sandbox/26/execroot/__main__ && \
  exec env - \
    PATH=/home/ced/.local/bin:/home/ced/bin:/opt/jdk/bin:/opt/cmake/bin:/usr/local/bin:/usr/bin:/home/ced/bin:/usr/local/sbin:/usr/sbin \
    PWD=/proc/self/cwd \
  /opt/mpich-3.2.1/bin/mpicxx -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer '-std=c++0x' -MD -MF bazel-out/k8-fastbuild/bin/plato/util/_objs/thread_local_object/thread_local_object.pic.d '-frandom-seed=bazel-out/k8-fastbuild/bin/plato/util/_objs/thread_local_object/thread_local_object.pic.o' -fPIC -iquote . -iquote bazel-out/k8-fastbuild/bin -Ibazel-out/k8-fastbuild/bin/3rd/glog/_virtual_includes/glog -isystem plato/util -isystem bazel-out/k8-fastbuild/bin/plato/util -isystem 3rd/boost/include -isystem bazel-out/k8-fastbuild/bin/3rd/boost/include -O2 -Wall '-std=c++11' -fno-canonical-system-headers -Wno-builtin-macro-redefined '-D__DATE__="redacted"' '-D__TIMESTAMP__="redacted"' '-D__TIME__="redacted"' -c plato/util/thread_local_object.cc -o bazel-out/k8-fastbuild/bin/plato/util/_objs/thread_local_object/thread_local_object.pic.o)
Execution platform: @bazel_tools//platforms:host_platform

Use --sandbox_debug to see verbose messages from the sandbox: mpicxx failed: error executing command 
  (cd /home/ced/.cache/bazel/_bazel_ced/d3af6ab933838aa4af081ccbbafd9c2e/sandbox/processwrapper-sandbox/26/execroot/__main__ && \
  exec env - \
    PATH=/home/ced/.local/bin:/home/ced/bin:/opt/jdk/bin:/opt/cmake/bin:/usr/local/bin:/usr/bin:/home/ced/bin:/usr/local/sbin:/usr/sbin \
    PWD=/proc/self/cwd \
  /opt/mpich-3.2.1/bin/mpicxx -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer '-std=c++0x' -MD -MF bazel-out/k8-fastbuild/bin/plato/util/_objs/thread_local_object/thread_local_object.pic.d '-frandom-seed=bazel-out/k8-fastbuild/bin/plato/util/_objs/thread_local_object/thread_local_object.pic.o' -fPIC -iquote . -iquote bazel-out/k8-fastbuild/bin -Ibazel-out/k8-fastbuild/bin/3rd/glog/_virtual_includes/glog -isystem plato/util -isystem bazel-out/k8-fastbuild/bin/plato/util -isystem 3rd/boost/include -isystem bazel-out/k8-fastbuild/bin/3rd/boost/include -O2 -Wall '-std=c++11' -fno-canonical-system-headers -Wno-builtin-macro-redefined '-D__DATE__="redacted"' '-D__TIMESTAMP__="redacted"' '-D__TIME__="redacted"' -c plato/util/thread_local_object.cc -o bazel-out/k8-fastbuild/bin/plato/util/_objs/thread_local_object/thread_local_object.pic.o)
Execution platform: @bazel_tools//platforms:host_platform

Use --sandbox_debug to see verbose messages from the sandbox
In file included from plato/util/thread_local_object.h:38:0,
                 from plato/util/thread_local_object.cc:24:
bazel-out/k8-fastbuild/bin/3rd/glog/_virtual_includes/glog/glog/logging.h:86:10: fatal error: gflags/gflags.h: No such file or directory
 #include <gflags/gflags.h>